static/script/devices/mediaplayer/html5.js

```diff
@@ -369,13 +370,11 @@ require.def(
             },
 
             _onMetadata: function() {
-                this._readyToPlayFrom = true;
-                if (this._waitingToPlayFrom()) {
-                    this._deferredPlayFrom();
-                }
+                this._metadataLoaded();
             },
 
             _exitBuffering: function () {
+                this._metadataLoaded();
```

Comment:
I'm confused. If we are STOPPED and then we do a playFrom(whatever), play() does not get called on the media element until after we have loaded metadata. We wont know we've loaded metadata until we get exitBuffering, which will only happen once we're playing. So won't a 'start playback at 5 mins' test on the HiSense with this new code never start playing back at all? I'm probably missing something :-)